### DISCUSSION TITLE
I don't understand its description

### DISCUSSION THREAD

**Discussion by GabrielOnDelphi** [alefragnani/delphi-james#27] [Q&A] [answered]
What the program does?
I can switch between projects also from the Project Manager. So, is this a better Project Manager?
Screenshot?

Thanks.

**Comment by alefragnani** [comment, marked as answer]
Hi @GabrielOnDelphi ,

For sure you can switch between projects using Project Manager, just like using `Open Project` command, ou double clicking in any `.dpr` / `.dproj` on Windows Explorer. And, if for your projects, once you switch, Delphi is _ready to use_, to compile, debug and run those projects, you don’t need James.

On the other hand, if, after you switch/open a project, _you see yourself manually installing components and updating Delphi settings_ (like Environment Variables), James will help you. 

You will face such issues if your projects don’t share the same set of third party components, or have more than one version of the same component. This makes each project using a _unique set of components_. In this case, James helps you to manage this _switches_.

But, if you don’t face these issues, James isn’t needed.

Hope this helps
